### DISCUSSION TITLE
Sample code for "org.openssl.winstore://"

### DISCUSSION THREAD

**Discussion by psc161** [openssl/openssl#23340] [Q&A]
Hello,
where can I find sample code for loading the root certificates from the Windows certificate store ("org.openssl.winstore://") using the openssl api functions?

**Comment by marsupilami79** [comment]
Hello,

I didn't use C to program this. For me it worked to call ``SSL_CTX_load_verify_store`` with the winstore URI before starting the TLS handshake.

Best regards,

Jan
